magit-annex.el

```diff
@@ -145,6 +145,10 @@ program used to open the unused file."
   :package-version '(magit-annex . "1.3.0")
   :type 'boolean)
 
+(defcustom magit-annex-file-wildcard-regexp "[[*?]"
+  "Regular expression that matches file names which should have wildcards expanded."
+  :type 'regexp)
```

Comment:
If we stick with adding an option, this should get a :package-version value.  (The next release with be 1.8.0.)